lib/cmds/config_cmds/remove.js

```diff
@@ -61,9 +58,18 @@ export const removeHandler = async (argv) => {
     if (argv.activeSpaceId) {
       delete options['activeSpaceId']
     }
+    if (argv.activeEnvironmentId) {
```

Comment:
Maybe we can have something like 
```js
const contextKeys = ['activeEnvironmentId', 'proxy', 'host', 'rawProxy']
contextKeys.forEach(key => {
	if(argv[key]) {
      delete options[key]
	}
})
```
This way when we add support for a new config property we only update the `contextKeys` array

Reply:
👍